python/mlc_chat/interface/convert_weight.py

```diff
@@ -51,6 +53,7 @@ def _device_to_str(device: Device) -> str:
 
 
 def _convert_args(args: ConversionArgs) -> None:  # pylint: disable=too-many-locals
+    pre_shards_num = os.getenv("PRESHARD_NUM")
```

Comment:
shall we use some cli argument instead of env var?

Reply:
Great question! Pre-sharding is intended for internal use only. To avoid confusing MLC users, I prefer to conceal this argument outside the regular pipeline.

Reply:
maybe MLC_INTERNAL_PRESHARD_NUM?

Reply:
Sounds good! updated

Reply:
> Pre-sharding is intended for internal use only

Why is that? @LeshengJin

Reply:
so, if I wanna use it just set MLC_INTERNAL_PRESHARD_NUM? dont need to re-compile config lib  weight or sth?